arcor2_AREditor/Assets/TABLET/Scripts/GUI/WalktroughManager.cs

```diff
@@ -0,0 +1,489 @@
+            HighlitedButton = null,
+        },
+
+        new WalktroughStep {
+            Order = 4,
+            Progress = -1,
+            PrimaryText = "Nice! \n The highlighted crosshair is your main selector.",
+            SecondaryText = "Tap on the next button.",
+            Tip = "Aim it on the object you want to work with.",
+            //Tip = "Left menu contains tools and functionalities and right menu holds objects which are placed in the scene.",
+            HighlitedButton = null,
+        },
+
+        
+
+        new WalktroughStep {
+            Order = 5,
+            PrimaryText = "Using the ADD button allows you to add objects.",
+            SecondaryText = "We are going to add new virtual object representing the robot",
+            Tip = "",
+            HighlitedButton = null,
+        },
+
+        new WalktroughStep {
+            Order = 6,
+            PrimaryText = "This button is used to add predefined objects like robots.",
+            SecondaryText = "Let's add the robot.",
+            Tip = "Second button allows you to create collision objects of basic shapes.",
+            HighlitedButton = null,
+        },
+
+        new WalktroughStep {
+            Order = 7,
+            PrimaryText = "Right menu contains objects in scenes or detailed functionality options .",
+            SecondaryText = "Now we can choose the robot or device that we have available.",
+            Tip = "",
+            HighlitedButton = null,
+        },
+
+        new WalktroughStep {
+            Order = 8,
+            PrimaryText = "",
+            SecondaryText = "",
+            Tip = "Enter information in the input fields and create an object.",
+            HighlitedButton = null,
+        },
+
+        new WalktroughStep {
+            Order = 9,
+            Progress = -1,
+            PrimaryText = "Great! \n Renaming, deleting, settings and other actions are available in Utility menu.",
+            SecondaryText = "You can now position the virtual object to match the real robot.",
+            Tip = "",
+            HighlitedButton = null,
+        },
+
+        new WalktroughStep {
+            Order = 10,
+            PrimaryText = "This tool allows you to rotate and position the object you are aiming at.",
+            SecondaryText = "Let's pick the movement tool.",
+            Tip = "Moving pricniples are the same throughout AREditor.",
+            HighlitedButton = null,
+        },
+
+
+
+
+
+        new WalktroughStep {
+            Order = 11,
+            PrimaryText = "When you aim at one of the arrows, you can move or rotate the object, by using slider on the right.\n",
+            SecondaryText = "Three arrows represent three axis of movement and rotation. Try to use slider and tap next.",
+            Tip = "You can switch rotation or positioning under slider.",
+            HighlitedButton = null,
+        },
+
+        // lepsie popisat slajder s ktorym sa hybe a co treba urobit pri tom zarovnani
```

Comment:
Please, convert comments to English and add `TODO: ` in case of tasks. Or create issues, or remove them :)